README.md

```diff
@@ -1245,25 +1244,27 @@ Example: https://github.com/aquasecurity/trivy-ci-test/blob/master/Dockerfile
 NAME:
   trivy - A simple and comprehensive vulnerability scanner for containers
 USAGE:
-  main [options] image_name
+  trivy [options] image_name
 VERSION:
-  0.0.15
+  0.1.6
 OPTIONS:
   --format value, -f value    format (table, json) (default: "table")
   --input value, -i value     input file path instead of image name
   --severity value, -s value  severities of vulnerabilities to be displayed (comma separated) (default: "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL")
   --output value, -o value    output file name
   --exit-code value           Exit code when vulnerabilities were found (default: 0)
   --skip-update               skip db update
+  --only-update value         update db only specified distribution (comma separated)
   --reset                     remove all caches and database
   --clear-cache, -c           clear image caches
-  --quiet, -q                 suppress progress bar
+  --quiet, -q                 suppress progress bar and log output
+  --no-progress               suppress progress bar
   --ignore-unfixed            display only fixed vulnerabilities
   --refresh                   refresh DB (usually used after version update of trivy)
   --auto-refresh              refresh DB automatically when updating version of trivy
   --debug, -d                 debug mode
-  --vuln-type value           comma-separated list of vulnerability types (os,library)
-  --cache-dir value           cache directory (default: "/path/to/cache")
+  --vuln-type value           comma-separated list of vulnerability types (os,library) (default: "os,library")
+  --cache-dir value           cache directory (default: "~/.cache/trivy")
```

Comment:
This default value depends on the machine running Trivy. In my environment (macOS), `default: "/Users/knqyf263/Library/Caches/trivy"`.

Reply:
OK

Return to original format